samekshareddy_Smart-SparkNotes_UI_84.py

```
@@ -0,0 +1,65 @@
+from werkzeug import secure_filename
+app = Flask(__name__, static_folder='static/search/', static_url_path='')
+import json
+import sys
+import os
+import logging
+
+import coffeescript
+
+from PDFParser import PDFParser
+
+UPLOAD_FOLDER = '/tmp/'
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)